Last-Update: 2012-09-01
Author: أحمد المحمودي (Ahmed El-Mahmoudy) <aelmahmoudy@sabily.org>
Reviewed-by: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Description: Add /usr/lib to sys.path for xpra script and /usr/lib/parti
 to sys.path for parti script.

--- a/scripts/parti
+++ b/scripts/parti
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 
 import sys
+sys.path.append("/usr/lib/parti")
 import parti.scripts.main
 
 parti.scripts.main.main(sys.argv)
--- a/scripts/parti-repl
+++ b/scripts/parti-repl
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 
 import sys
+sys.path.append("/usr/lib/parti")
 import parti.scripts.repl
 
 parti.scripts.repl.main(sys.argv)
--- a/scripts/xpra
+++ b/scripts/xpra
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 
 import sys
+sys.path.append("/usr/lib")
 import xpra.scripts.main
 
 sys.exit(xpra.scripts.main.main(__file__, sys.argv))
--- a/scripts/xpra_launcher
+++ b/scripts/xpra_launcher
@@ -1,5 +1,7 @@
 #!/usr/bin/env python
 
+import sys
+sys.path.append("/usr/lib")
 from xpra.scripts.client_launcher import main
 
 main()
